packages/dynamic-import-vars/README.md

```diff
@@ -134,7 +134,11 @@ function __variableDynamicImportRuntime__(path) {
     case './locales/nl-NL.js':
       return import('./locales/nl-NL.js');
     default:
-      throw new Error('Unknown variable dynamic import: ' + path);
+      return new Promise((resolve, reject) => {
+        queueMicrotask(() => {
+          reject(new Error("Unknown variable dynamic import: " + path))
```

Comment:
Sorry not reviewed this in detail, but wondering if `queueMicrotask` is supported in most places or if we should use something a bit older instead to achieve this?

Would a standard `return Promise.reject()` not work?

Reply:
It would not. The actual code falls back to `setTimeout`.

Reply:
Looked in a bit more detail.

What wouldn’t work with `return Promise.reject(new Error("Unknown variable dynamic import: " + path))` work?

The return is always a promise then which would resolve/reject async. I understand why the sync throw  before was not great

Reply:
The code actually already is using `Promise.reject`. The documentation just doesn't match. Please run this with your dev tools open and `Pause on exception` enabled to see what this fixes.

https://jsfiddle.net/g6u4femL/

Note that even though both promises are caught, it will still break on `promiseA`.

Reply:
Ah ok thanks for explaining.

Seems strange there’s not a simpler way but 👍

Reply:
Feels like this is something dev tools should handle? If you’re pausing on uncaught exceptions I wonder why it doesn’t wait a tick internally until the point when it actually rejects 🤔